.gitignore

```diff
@@ -24,6 +24,7 @@ src/transfer.sh
 src/configs
 redis.ds
 src/redis.conf
```

Comment:
```suggestion
```

Reply:
Although, now I see this is being fixed here: https://github.com/valkey-io/valkey/pull/29/files. So maybe best to just wait for that to get merged.

Reply:
Sure, it works for me. I would wait till https://github.com/valkey-io/valkey/pull/29/files merged and close this PR. Thanks.